examples/step-68/step-68.cc

```diff
@@ -0,0 +1,745 @@
+// a collection of points with some attached properties (e.g., an id) floating
+// on a parallel::distributed::Triangulation. The methods and classes in the
+// namespace Particles allows one to easily implement Particle-In-Cell methods
+// and particle tracing on distributed triangulations
+#include <deal.II/particles/particle_handler.h>
+
+// We import the particles generator
+// which allow us to insert the particles. In the present step, the particle
+// are globally inserted using a non-matching hyper-shell triangulation
+#include <deal.II/particles/generators.h>
+
+// Since the particles do not form a triangulation, they have their
+// own specific data out class which will enable us to write them
+// to commonly used parallel vtu format
+#include <deal.II/particles/data_out.h>
+
+
+// This step uses parallel vector to interpolate the velocity field
+// at the position of the particles. This step supports the use of both
+// Trilinos and PETSC distributed vectors
+#define FORCE_USE_OF_TRILINOS
+
+namespace LA
+{
+#if defined(DEAL_II_WITH_PETSC) && !defined(DEAL_II_PETSC_WITH_COMPLEX) && \
+  !(defined(DEAL_II_WITH_TRILINOS) && defined(FORCE_USE_OF_TRILINOS))
+  using namespace dealii::LinearAlgebraPETSc;
+#  define USE_PETSC_LA
+#elif defined(DEAL_II_WITH_TRILINOS)
+  using namespace dealii::LinearAlgebraTrilinos;
+#else
+#  error DEAL_II_WITH_PETSC or DEAL_II_WITH_TRILINOS required
+#endif
+} // namespace LA
+
+#include <cmath>
+#include <iostream>
+
+namespace Step68
+{
+  using namespace dealii;
+
+  // @sect3{Run-time parameter handling}
+
+  // Similarly to what is done in step-60, we set up a class that holds
+  // all the parameters of our problem and derive it from the ParameterAcceptor
+  // class to simplify the management and creation of parameter files.
+  //
+  // The ParameterAcceptor paradigm requires all parameters to be writable by
+  // the ParameterAcceptor methods. In order to avoid bugs that would be very
+  // difficult to track down (such as writing things like `time = 0` instead of
+  // `time == 0`), we declare all the parameters in an external class, which is
+  // initialized before the actual `ParticleTracking` class, and pass it to
+  // the main class as a `const` reference.
+  //
+  // The constructor of the class is responsible for the connection between the
+  // members of this class and the corresponding entries in the
+  // ParameterHandler. Thanks to the use of the
+  // ParameterHandler::add_parameter() method, this connection is trivial, but
+  // requires all members of this class to be writeable.
+  class ParticleTrackingParameters : public ParameterAcceptor
+  {
+  public:
+    ParticleTrackingParameters();
+
+    // This class consists largely of member variables that
+    // describe the details of the particle tracking simulation and its
+    // discretization. The following parameters are about where output should
+    // land, the spatial discretization of the velocity (the default is $Q_1$),
```

Comment:
land?

Reply:
I think this is not yet addressed, maybe `where output should be written` or `stored`?